packages/astro/test/fixtures/lit-element/src/components/my-element.js

```diff
@@ -3,9 +3,26 @@ import { LitElement, html } from 'lit';
 export const tagName = 'my-element';
 
 export class MyElement extends LitElement {
+  static properties = {
+    bool: {type: Boolean},
+    str: {type: String},
+    obj: {type: Object},
+  }
+
+  constructor() {
+    super();
+    this.bool = true;
+    this.str = 'not initialized';
+    this.obj = {data: null};
+    // not a reactive property
+    this.foo = 'not initialized';
+  }
   render() {
     return html`
       <div>Testing...</div>
+      <div id="bool">${this.bool}</div>
```

Comment:
I would write an expression that has a different output when the value is a bool vs the string "false". Something like `this.bool === false ? 'A' : 'B'`